### ISSUE TITLE
database_setup, todo toType handle: VoidType

### ISSUE THREAD

**Issue by Zeatynis** [abaplint/transpiler#762]
Hi,
I need help in this one. How should I know what is wrong and how to fix that?
I get:
```
Transpiler CLI
Using config: abap_transpile.json
Clone: https://git....com/abap/rest-engine
184 files added from lib
Clone: https://github.com/open-abap/open-abap
283 files added from lib
292 files added
689 files skipped
Building
database_setup, todo toType handle: VoidType
```

**Comment by larshp** [comment]
there is some type which is unknown in your setup, configure abaplint to strictly check everything, ie set the `errorNamespace` to everything

as transpiling requires complete knowledge about types

**Comment by Zeatynis** [comment]
I have it like this:
`"errorNamespace": "^(/HCR/|Z|Y|LT?CL_|LCL_|TY_|LIF_)",`

**Comment by larshp** [comment]
yea, change it to
`"errorNamespace": ".",`
to strictly syntax check everything

**Comment by Zeatynis** [comment]
same...

**Comment by Zeatynis** [comment]
Isn't there any way to check which type is unknown? Any logs/verbose output?

**Comment by larshp** [comment]
example verbose output = https://abaplint.app/stats/abapGit/abapGit/void_types, the linter is linting, not reporting

**Comment by larshp** [comment]
but anyhow, then I guess there is something wrong in the setup you use, and/or configuration

**Comment by Zeatynis** [comment]
> example verbose output = https://abaplint.app/stats/abapGit/abapGit/void_types, the linter is linting, not reporting

oh, right. I can't use this app 😢I'm nailed into using GitLab. I suppose you won't tell me how to achieve this functionality manually? 😝

**Comment by larshp** [comment]
well, you should not need to, the linter will(should) tell you

did you disable any of the minimal rules mentioned in https://github.com/abaplint/abaplint/blob/main/docs/getting_started.md ? like `unknown_types` ?

**Comment by Zeatynis** [comment]
No, I have all of them enabled and theirs severity is set to errors.

**Comment by Zeatynis** [comment]
Here I uploaded my config: https://github.com/Zeatynis/abaplint-issue
I use npm unit and that causes the issue.

**Comment by larshp** [comment]
@Zeatynis try again, 2.0.46, it should give a better error message

**Comment by larshp** [comment]
add to open-abap, the DTEL is released https://abapedia.org/steampunk-2205-api/rfcdest.dtel.html

**Comment by Zeatynis** [comment]
What are these steampunk apis?
https://github.com/abapedia/steampunk-2205-api

Shall I use OpenAbap or Steampunk as dependenciy?

**Comment by larshp** [comment]
if you want to run the code use open-abap

if you want to syntax check against steampunk use the abapedia, it does not contain any implementation code, ie. it cannot be run
